bciers/yarn.lock

```diff
@@ -6247,7 +6247,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@types/node@npm:*, @types/node@npm:^20.11.25, @types/node@npm:^20.8.4":
+"@types/node@npm:*, @types/node@npm:^20.8.4":
   version: 20.14.2
   resolution: "@types/node@npm:20.14.2"
   dependencies:
```

Comment:
Looks like something still keeps references to node 20 types

Reply:
We can bump it to version `6.0.1` instead of the latest version, because version 7 has breaking changes and would also require upgrading the MUI package: https://github.com/viclafouch/mui-tel-input/releases/tag/v7.0.0